### DISCUSSION TITLE
Documentation / library for a flask based server integration.

### DISCUSSION THREAD

**Discussion by KedarKshatriya** [Web3Auth/Web3Auth#495] [Help]
Hey People,
Any method or idea on how I could integrate web3auth in a Flask API Backend (Python) environment?
Also, It would be fantastic if you could provide me with some links to a demo or any codebase which have done something similar.

Cheers :-)

**Comment by shahbaz17** [comment]
Hi @KedarKshatriya 

As of now, there's no Flask/Python SDK for Web3Auth. We will be bringing more SDK support in the future.

Thanks
